src/zcl_aff_test_types.clas.abap

```diff
@@ -68,7 +68,7 @@ CLASS zcl_aff_test_types DEFINITION
       END OF struc_link_wrong_type.
 
     TYPES:
-    "! $pattern '[a-Z]*'
+    "! $pattern '\\n\\t\\r[a-z]*\\"'
```

Comment:
I would add an example with \n, \t, \r

Reply:
ok, I'll add an additional example for escape sequences